html-and-the-web_div-salad/index.html

```diff
@@ -0,0 +1,83 @@
+<!--
+This page looks so confusing.
+It would be much better to work with semantic HTML to structure this code.
+Let's fix this mess!
+(There are hints to help you out below some elements)
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Basics Excercise</title>
+    <meta charset="UTF-8" />
+  </head>
+  <!--someone forgot to wrap the content of our webpage with an HTML element,
+	which one was it again?-->
+  <body>
+    <header>
+      <h1>Jane Doe</h1>
+      <!--Which elements could be used to link to parts of the webpage,
+		which appear below?-->
+      <nav id="nav">
+        <a href="#about-me">About Me</a>
+        <a href="#contact">Contact</a>
+        <a href="#skills">Skillset</a>
+      </nav>
+    </header>
+    <!--Looks like the following part is the central content of our webpage.-->
+    <main>
+      <section id="about-me">
+        <h3>About me</h3>
+        <p>
+          Hi there! Welcome to my personal website. Here is gathered the most
+          important information about me. If you want to work with me or get to
+          know me, please don't hesitate to get in contact.
+        </p>
+        <!--What element could be used to combine an image with a descriptive caption?-->
+        <section>
+          <img
+            src="https://source.unsplash.com/random/400x300?fruit"
+            alt="a random picture"
+          />
+          <p>
+            A random picture, because I would like to maintain my privacy and
+            let my code speak for me, instead of images.
+          </p>
+        </section>
+      </section>
+
+      <section id="contact">
+        <h3>Contact</h3>
+        <!--When clicking on these element, nothing happens! But I want to be redirected to the other websites. Why is it not working?-->
+        <a href="https://github.com/spiced-academy">GitHub</a>
+        <a href="https://de.linkedin.com/">LinkedIn</a>
+        <a href="https://codesandbox.io/">CodeSandbox</a>
+      </section>
+
+      <section id="skills">
+        <h3>Skillset</h3>
+        <!--This part could use some bulletpoint to be more structured-->
+        <ul>
+          <ol>
```

Comment:
Inside an `un`or `ol` must be a `li`. Please review and correct.

Reply:
I fixed it